layouts/integrations/single.html

```diff
@@ -1,4 +1,12 @@
 {{ define "main" }}
+    {{ $media := slice }}
+
+    {{ if and (.Params.tile) (.Params.tile.media) }}
+        {{ $media = .Params.tile.media }}
+    {{ else if .Params.media }}
+        {{ $media = .Params.media }}
+    {{ end }}
```

Comment:
maybe we can use default instead i think something like this might work `{{ $media := .Params.tile.media | default .Params.media | default "" }}`